src/Examine.Test/Examine.Core/ValueSetTests.cs

```diff
@@ -0,0 +1,129 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using NUnit.Framework;
+
+namespace Examine.Test.Examine.Core
+{
+    [TestFixture]
+    public class ValueSetTests
+    {
+
+        [Test]
+        public void TestYieldConvertsSingleObjectsToEnumerable()
```

Comment:
I know the rest of Examine tests don't follow a standard naming convention practice but I would like us to adopt Given-When-Then Gerkin format moving forward (https://martinfowler.com/bliki/GivenWhenThen.html)

For example, these could be:

* Given_SingleValues_When_Yielding_ThenConvertsToEnumerable
* Given_MultiValues_When_Yielding_ThenConvertsToEnumerable
* Given_MixedValues_When_Yielding_ThenConvertsToEnumerable

Reply:
Done